addons/optionsmenu/XEH_preInit.sqf

```diff
@@ -15,4 +15,50 @@ GVAR(serverSideValues) = [];
 GVAR(categories) = [];
 GVAR(currentCategorySelection) = 0;
 
+recomp = {
```

Comment:
This is just for development, recompiles all funcs before opening menu.
Makes testing very fast.
Will remove when PR is ready.